.babelrc.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @HACK: Babel doesn't support using modules as your `babelrc`.
```

Comment:
The issue with this is that node-register doesn't support imports. So none of the npm scripts will work because the import statements won't be transpiled. Use your solution of reading and parsing the `.babelrc`